res/images/templates/ic_template_keyboard_mapping_sheet.svg

```diff
@@ -11,13 +11,13 @@
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
    sodipodi:docname="ic_template_keyboard_mapping_sheet.svg"
-   inkscape:version="0.48.2 r9819"
```

Comment:
This file still references "Mixxx 1.11" multiple times. Could you please update this to "Mixxx 2.2"? Maybe we should remove the version numbers entirely, otherwise we forget to update them with every new version.

Reply:
I removed the version numbers.
Also, I resized the SVG to ease PNG export by simply selecting "Entire Page" and exporting to 1280px width